src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/ExistingRows.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Tr } from '../../../../generic/Table/table'
+import { PopupTd, PopupTdForRadio } from '../ImageAnnotationModal.styles'
+import { Select } from '../../../../generic/form'
+import {
+  imageClassificationPointPropType,
+  imageClassificationResponsePropType,
+} from '../../../../../App/mermaidData/mermaidDataProptypes'
+
+const isClassified = ({ is_unclassified, annotations }) =>
+  !is_unclassified && annotations.length > 0
+
+const isAClassifierGuessOfSelectedPoint = (annotations, benthic_attribute, growth_form) =>
+  annotations.some(
+    (annotation) =>
+      !!annotation.is_machine_created &&
+      annotation.benthic_attribute === benthic_attribute &&
+      annotation.growth_form === growth_form,
+  )
+
+const isAlreadyPushed = (acc, value) => acc.some((option) => option.value === value)
+
+const ExistingRows = ({
+  selectedPoint,
+  dataToReview,
+  setDataToReview,
+  selectedRadioOption,
+  setSelectedRadioOption,
+  getBenthicAttributeLabel,
+  getGrowthFormLabel,
+}) => {
+  const existingRowDropdownOptions = dataToReview.points.reduce((acc, currentPoint) => {
+    const { benthic_attribute, growth_form } = currentPoint.annotations[0]
+    const value = benthic_attribute + '_' + growth_form
+    const label = `${getBenthicAttributeLabel(benthic_attribute)} ${getGrowthFormLabel(
+      growth_form,
+    )}`
+
+    if (
+      isClassified(currentPoint) &&
+      !isAlreadyPushed(acc, value) &&
+      !isAClassifierGuessOfSelectedPoint(selectedPoint.annotations, benthic_attribute, growth_form)
+    ) {
+      acc.push({ label, value })
+    }
+
+    return acc
+  }, [])
+
+  const [selectedExistingRow, setSelectedExistingRow] = useState(() => {
+    const rowKeyForPoint =
+      selectedPoint.annotations[0].benthic_attribute +
+      '_' +
+      selectedPoint.annotations[0].growth_form
+    const isPointInARow = existingRowDropdownOptions.some((row) => rowKeyForPoint === row.value)
+    return isPointInARow ? rowKeyForPoint : ''
+  })
+
+  const addExistingAnnotation = (existingAnnotation) => {
+    const [benthic_attribute, growth_form] = existingAnnotation.split('_')
+
+    // Only want classifier guesses, and want to them unconfirmed.
+    const resetAnnotationsForPoint = selectedPoint.annotations.reduce((acc, currentAnnotation) => {
```

Comment:
would prefer to not directly mutate the `currentAnnotation` object  and make a copy instead